packages/next/src/server/app-render/create-component-tree.tsx

```diff
@@ -705,6 +711,7 @@ async function createComponentTreeInternal({
         <OutletBoundary>
           <MetadataOutlet ready={getViewportReady} />
           <MetadataOutlet ready={getMetadataReady} />
+          {metadataOutlet}
```

Comment:
I don't think we need to put it into a variable and could just render `<StreamingMetadataOutlet />` here.

Reply:
good catch it should be inside layout router under the error boundaries

Reply:
ignore my previous reply, I read it wrong